### ISSUE TITLE
Change Texts

### ISSUE THREAD

**Issue by dodo77** [Hyphen-ated/DJBot#42]
Hello, I'd like to change the text witch the bot should write. But I can't change this. I've tried it with the sourcecode but nothing did work. @Hyphen-ated

**Comment by Hyphen-ated** [comment]
Have you successfully run the bot without changing the source code?

Have you ever programmed in Java before, and do you have a development environment set up?

**Comment by dodo77** [comment]
Yes, I used it without changing the code, nice bot by the way.

But if i try ti modify there a lot of errors in the files with jdk 1.8

**Comment by Hyphen-ated** [comment]
well post the errors you're getting. what do you honestly expect me to be able to do with the information you've provided?

**Comment by dodo77** [comment]
Yes I know. But there are more than 100 errors in 1 File. I don't know what's wrong.

How exactly does it works with the source files? Hot wo execute / compile, special program?

I've tried diffrent things but it doesn't work. There are always more than 100 errors and I think I made something wrong.

**Comment by Hyphen-ated** [comment]
have you tried reading what the errors say

**Comment by dodo77** [comment]
Ok. There are many Errors like "canmot resolve method" or "cannot resolve symbol". Also in the DjRescource there is a "cannot resolve symbol path", "cannot resolve symbol get", "cannot resolve method debug"

Is it possible, that I have to download libraries? And can you tell me wich libraries I need? Are there other things I need for customizing the tool or is it enought to download the DJBot-master?

**Comment by dodo77** [comment]
What do you think?

**Comment by Hyphen-ated** [comment]
Yeah it sounds like you're missing the libraries. This project uses a build tool called "maven" which handles downloading them automatically. (if you ever see a project with a file called pom.xml at the root, that means it uses maven.)

If you're using any of the popular java IDEs, they should have maven support built into them. If you're not, I recommend downloading intellij idea community edition. Then do file->new->project from existing sources, and point it to djbot's pom.xml

There's a few more instructions for how to run the bot from source in the file contributing.md

**Comment by dodo77** [comment]
Oh, now I think it works. Thanks 👍 
Is it possible to make this as .jar to execute like the "reqular" tool?

**Comment by Hyphen-ated** [comment]
instructions for that are in contributing.md

**Comment by dodo77** [comment]
I don't konw realy how to build it with these arguments.
